### ISSUE TITLE
Unable to load saved requests in mock lambda test tool

### ISSUE THREAD

**Issue by tsaideepak7** [aws/aws-lambda-dotnet#1630]
### Describe the bug

I have few saved requests in my mock test lambda tool, and with the verion 0.15.1, the requests are not loading,.
I notice that requests related to blazor are failing. (I'm using a simple plain lambda Fn using Annotations framework)

### Expected Behavior

The requests should be loaded

### Current Behavior

Requests are not loaded

### Reproduction Steps

Run Mock lambda test tool
Choose any saved request from the dropdown

### AWS .NET SDK and/or Package version used

Amazon.Lambda.TestTool-6.0 0.15.1

### Targeted .NET Platform

.Net Core 6

### Operating System and version

Windows 10

**Comment by ashishdhingra** [comment]
@tsaideepak7 Good morning. Somehow, I'm unable to reproduce the issue. Here are the steps that I followed:
- Installed Lambda Test Tool 6.0 version `0.14.1` using command ` dotnet tool install --global Amazon.Lambda.TestTool-6.0 --version 0.14.1`.
- Created a Lambda project targeting .NET 6.0. (I already had one with the below `launchSettings.json`)
  ```JSON
  {
    "profiles": {
      "Mock Lambda Test Tool": {
        "commandName": "Executable",
        "commandLineArgs": "--port 5050",
        "workingDirectory": ".\\bin\\$(Configuration)\\net6.0",
        "executablePath": "%USERPROFILE%\\.dotnet\\tools\\dotnet-lambda-test-tool-6.0.exe"
      }
    }
  }
  ```
- Debugged using Visual Studio and launched Lambda Test tool.
- Selected `API Gateway V2 HTTP API` example request from drop down and saved it as `APIGatewayHTTPV1Test`. This created `APIGatewayHTTPV1Test.json` under `<<ProjectDirectory>>\.lambda-test-tool\SavedRequests`.
- Upgraded to latest version `0.15.1` of Lambda Test Tool using command `dotnet tool update --global Amazon.Lambda.TestTool-6.0`.
-  Debugged using Visual Studio and launched Lambda Test tool. I could see the saved request in example requests dropdown.

Please check if you see any errors (share the same if any) in the Visual Studio console debug window, OR, have saved request JSON file under `<<ProjectDirectory>>\.lambda-test-tool\SavedRequests` (although you mentioned that downgrading to version `0.14.1` works for you).

Thanks,
Ashish

**Comment by tsaideepak7** [comment]
I was getting that error with 0.15.0, which i wrongly mentioned, it seemed to be fixed in 0.15.1, hence closing
